docs/src/modules/branding/CommunitySayCard.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Box, Avatar, Typography } from '@material-ui/core';
+
+interface CommunitySayCardProps {
+  name: string;
+  id: string;
+  description: any;
+  avatar: string;
+  uniqueKey: number;
+}
+export default function CommunitySayCard(props: CommunitySayCardProps) {
+  const { name, id, description, avatar, uniqueKey } = props;
+  return (
+    <Box
+      sx={{
+        bgcolor: 'greyF3',
+        px: { xs: 4, lg: 5 },
+        py: 5,
+        pt: 5.8,
```

Comment:
I think that it signals we don't care about the spacing unit, either we align it on 5.5 or 6. 

x all the other cases

Reply:
Updated the spacings. Now they are either  `x % 2 === 0` or `x % 5 === 0`